deployment/templates/rolebinding.yaml

```diff
@@ -12,4 +12,7 @@ roleRef:
   name: view
 subjects:
 - kind: ServiceAccount
-  name: lodestar-status
\ No newline at end of file
+  name: lodestar-status
+{{- if not .Values.development }}
+  namespace: {{ .Values.deploymentNamespace }}
```

Comment:
I believe we can use `{{ .Release.Namespace }}` instead. This works when I test with `helm template ....` locally but not entirely sure how it will behave in ArgoCD. I'd like to get input from @tylerauerbeck here as well.
See https://helm.sh/docs/chart_template_guide/builtin_objects/

Reply:
I believe I've seen a number of other projects use this in a similar manner. So it _should_ work.

Reply:
Thanks guys.  I updated to use `{{.Release.Namespace.}}`